### DISCUSSION TITLE
lauching t3.small instance

### DISCUSSION THREAD

**Discussion by alekhya9999** [learndevopsonline/learndevopsonline#33] [Q&A] [answered]
Hi @r-devops,Please clarify shall i move to launch this instance

![Screenshot_20230214_190842](https://user-images.githubusercontent.com/122102519/218755091-e6521be9-f7c6-4c98-8644-d1ed730cee5a.png)

**Comment by ldo4github** [comment, marked as answer]
@alekhya9999 While selecting Resource types do not select the Elastic GPUs tag, then it will work

**Reply by alekhya9999** [reply]
Yes,Tq For response @ldo4github 
It will launch if i deselect that one,Just to know clearly if that make any changes further
